dbschema/src/test/java/org/folio/dbschema/ObjectMapperToolTest.java

```diff
@@ -62,4 +73,43 @@ void canReadSchema() throws Throwable {
       is(true));
     assertThat(dbSchema.getExactCount(), is(10000));
   }
+
+  @ParameterizedTest
+  @CsvSource({
+    " 2024-12-31T13:14:15.789+00:00, 2024-12-31T13:14:15.789+00:00",
+    " 0000-01-01T00:00:00.000+00:00, 0000-01-01T00:00:00.000+00:00",
+    "+0000-01-01T00:00:00.000+00:00, 0000-01-01T00:00:00.000+00:00",
+    "+0000-12-31T23:59:59.999+00:00, 0000-12-31T23:59:59.999+00:00",
```

Comment:
Will all dates either have plus in front of them or a space? Or could some have no space and no plus?

Reply:
> CsvSource trims whitespace at the beginning and the end of the value: https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-sources-CsvSource
> 
> Do you want me to reformat using textBlock or single quotes?

No it's fine. I can see from the link that it trims by default. There's an option if you don't want to trim but I don't see a reason for that here.